modules/ztp-configuring-ipsec-using-ztp-and-siteconfig-for-mno.adoc

```diff
@@ -0,0 +1,159 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-advanced-install-ztp.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ztp-configuring-ipsec-using-ztp-and-siteconfig-for-mno_{context}"]
+= Configuring IPsec encryption for multi-node OpenShift clusters using {ztp} and SiteConfig resources
+
+You can enable IPsec encryption in managed multi-node clusters that you install using {ztp} and {rh-rhacm-first}.
+You can encrypt external traffic between pods and IPsec endpoints external to the managed cluster. All pod-to-pod network traffic between nodes on the OVN-Kubernetes cluster network is encrypted with IPsec in Transport mode.
```

Comment:
These 2 sentences incorrectly identify the type of traffic to be encrypted. I would reword them as follows:
(You may even consider mentioning North/South traffic which is OCP cluster ingress/egress traffic as opposed to East/West traffic which is OCP cluster intra node traffic)
You can encrypt traffic between the managed cluster and IPsec endpoints external to the managed cluster.
All network traffic between nodes on the OVN-Kubernetes cluster network is encrypted with IPsec in Transport mode.

Reply:
Updated as per the suggestion.